assets/high-contrast-canvas.css

```diff
@@ -8,33 +8,18 @@
   --context-pad-entry-background-color: var(--gray-10);
 }
 
-.dmn-drd-container,
-.bjs-container {
+.djs-container.bio-improved-canvas,
+.dmn-drd-container.bio-improved-canvas {
   background-color: var(--gray-10);
 }
 
-.dmn-drd-container {
+.dmn-drd-container.bio-improved-canvas {
   --dmn-definitions-background-color: white;
   --dmn-definitions-border-color: var(--gray-20);
 }
 
-/* Properties Panel */
-
-.bio-properties-panel-header {
-  background-color: white;
-}
```

Comment:
Can we safely get rid of this? Before the change, the top was white:

![image](https://github.com/camunda/improved-canvas/assets/21984219/1ffa705b-dc48-4e92-9ad7-3a533fcb4b21)

Now it is Grey
![image](https://github.com/camunda/improved-canvas/assets/21984219/30e97c6f-cb64-42bd-9298-3988458730b6)

Reply:
We might need to add namespaces to the properties panel container as well

Reply:
This is now adjusted by the web modeler itself.

Reply:
Also because we can't namespace it from inside the canvas and it needs to be applied conditionally.

Reply:
Good to go from my side then 👍